crates/terminal/src/lib.rs

```diff
@@ -129,6 +129,24 @@ macro_rules! ceprint {
     };
 }
 
+pub fn print_link(url: &str) {
+    if atty::is(atty::Stream::Stdout) {
+        print!("{}", link(url));
+    } else {
+        print!("{url}");
+    }
+}
+
+#[cfg(unix)]
+fn link(url: &str) -> String {
+    format!("\x1B]8;;{url}\x1B\\{url}\x1B]8;;\x1B\\")
```

Comment:
Is there a reason we're not using termcolor for this? Is there not support for what we're trying to achieve?

Reply:
yes by adding color would also help

Reply:
not an expert but can we use https://docs.rs/colored/latest/colored/

Reply:
or else we can us the color as raw hexadecimal format
which also works when not want to add more dependencies